deployment-guide.md

```diff
@@ -20,44 +20,27 @@ To be able to do that, your development environment shall already have Docker da
 Note, that the project itself depends on Golang 1.10+ and glide being available, but we packaged these dependencies into an automatically created builder container, so you don't have to worry about them!
 ### Building the binaries
 
-It is actually as easy as go get-ting the repository from GitHub, and executing the build_danm.sh script from the root of the project!
+It is actually as easy as cloning the repository from GitHub, and executing the build_danm.sh script from the root of the project!
+
 ```
-go get -d github.com/nokia/danm
-cd $GOPATH/src/github.com/nokia/danm
+git clone github.com/nokia/danm
+cd danm
 ./build_danm.sh
 ```
-This will first build the Alpine based builder container, mount the $GOPATH/src and the $GOPATH/bin directory into it, and invoke the necessary script to build all binaries inside the container.
-The builder container destroys itself once its purpose has been fulfilled.
 
-The result will be 6, statically linked binaries put into your $GOPATH/bin directory.
+The result will four container images:
 
-- danm
+- `danm-cni-plugins`: This image contains the core CNI plugins (`danm`, `fakeipam`) as well as the `macvtap`
```

Comment:
why macvtap, and not macvlan? do you use it for something? :)
we use macvlan, ipvlan and sriov mostly for control plane and data plane, so if we put some binaries inside IMO macvlan should bei ncluded instead / in addition to macvtap

Reply:
yes, sorry, brain fart/typo. Meant `macvlan`.